tests/chart/test_auth_sidecar.py

```diff
@@ -147,6 +146,52 @@ def test_auth_sidecar_resources_with_dag_server_enabled(self, kube_version):
             ],
         )
 
-        common_dagserver_sts_test_cases(docs, 1)
+        common_sts_test_cases(docs, 1, "release-name-dag-server", "StatefulSet")
         c_by_name = get_containers_by_name(docs[0])
         assert c_by_name["auth-proxy"]["resources"] == resources
+
+    def test_auth_sidecar_config_with_git_sync_server_enabled(self, kube_version):
+        """Test logging sidecar config with defaults"""
+        resources = {
+            "requests": {"cpu": 99.9, "memory": "777Mi"},
+            "limits": {"cpu": 66.6, "memory": "888Mi"},
+        }
+
+        volumeMounts = {
+            "mountPath": "/etc/nginx/nginx.conf",
+            "name": "nginx-sidecar-conf",
+            "subPath": "nginx.conf",
+        }
+
+        authSidecarServicePorts = {
+            "name": "auth-proxy",
+            "port": 8084,
+            "protocol": "TCP",
+            "targetPort": 8084,
+        }
+
+        docs = render_chart(
+            kube_version=kube_version,
+            values={
+                "gitSyncRelay": {"enabled": True, "repoFetchMode": "webhook"},
+                "authSidecar": {"enabled": True, "resources": resources},
+            },
+            show_only=[
+                "templates/git-sync-relay/git-sync-relay-deployment.yaml",
+                "templates/git-sync-relay/git-sync-relay-service.yaml",
+                "templates/git-sync-relay/git-sync-relay-auth-sidecar-configmap.yaml",
+            ],
+        )
+
+        common_sts_test_cases(docs, 3, "release-name-git-sync-relay", "Deployment")
```

Comment:
This applies to more than just `sts`
```suggestion
        common_pod_manager_test_cases(docs, 3, "release-name-git-sync-relay", "Deployment")
```